SMARTAGRIFOOD/wineTraceability/grapeTransportContainer/schema.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$schemaVersion": "0.0.8",
+    "modelTags": "",
+    "$id": "https://github.com/smart-data-models/incubated/SMARTAGRIFOOD/vineyardData/schema.json",
+    "title": "Smart Data Models - Grape Transport Container",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
+        },
+        {
+          "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+        },
+        {
+        "description": "This entity contains a harmonised description of a grape transport container (trailer, box of grapes, ...). ",
+        "properties": {
+          "type": {
+            "description": "Property. NGSI Entity Type: It has to be AgriGrapeTransportContainer",
+            "enum": [
+              "AgriGrapeTransportContainer"
+            ],
+            "type": "string"
+          },
+          "GRAI": {
+            "description": "Global Returnable Asset Identifier according to standard GS1 - Identifier GRAI. Up to 30 characters long, combining numeric and alphabetic characters.",
+            "type": "string",
+            "pattern": "^8003\\d{7,30}\\d{1}([a-zA-Z0-9]*)$"
+          },
+          "volumeEmpty": {
+            "description": "Property. Volume of container in cubic meters.",
+            "type": "number"
+          },
+          "weightEmpty": {
+            "description": "Property. Weight of empty container in kilograms.",
+            "type": "number"
+          },
+          "currentlyHoldsVariety": {
+            "description": "Grape varieties in trailer.",
```

Comment:
missing NGSI type. Possibly needed to add "Property. " at the beginning of the description